### PR TITLE
Fix ticket transfer & simplify element

### LINKED ISSUE #357: TicketTransfer bugs.
- [x] Popup doesn't dissappear when ticket is transferred.
- [x] Popup doesn't dissappear when ticket transfer is cancelled.
- [x] No succes notification happens for both actions.
- [x] When a ticket is already setup for transfer, the API sends `Ticket XX is already set up for transfer!` but the frontend just displays `The server was unable to process your request.`

### PR DESCRIPTION
**Pull request by martijnjanssen**
Fixes #357 
Reworked and simplified the transfer and cancel flows to improve on readability and functionality.

### CODE REVIEW

**Comment by MatthijsKok** [comment]
- [ ] "Email should be valid." check is done immediately.
- [ ] You can send a ticket to yourself. (API bug probably)

Otherwise great changes!

**Comment by MatthijsKok** [comment]
`"Email should be valid." check is done immediately.`
I think this needs to get fixed after all, Since the `label` already states `Email`, so it is not unclear to the user what to enter in the field.

**Comment by MatthijsKok** [comment]
I did not get any error when trying to transfer the ticket to myself. Maybe this is broken because the API now throws an exception?

**Comment by martijnjanssen** [comment]
The api fix was just pushed, this is not included in this PR.

File: src/lancie-my-area/my-area-orders.html [hunk 1]
```diff
@@ -57,7 +57,7 @@
       <div class="card-content">
         <template is="dom-if" if="[[tickets.length]]">
           <template is="dom-repeat" items="[[tickets]]" index-as="index">
-            <my-area-ticket-item username="{{username}}" ticket="[[item]]"></my-area-ticket-item>
+            <my-area-ticket-item user="[[username]]" ticket="[[item]]"></my-area-ticket-item>
```
Comment (TimvdLippe) [inline src/lancie-my-area/my-area-orders.html]: Shouldnt this be user?

**Pull request merged**